build.gradle.kts

```diff
@@ -11,17 +11,19 @@ configurations {
 }
 
 allOpen {
-  annotations("javax.persistence.Entity")
+  annotations("jakarta.persistence.Entity")
 }
 
+repositories {
+  maven { url = uri("https://repo.spring.io/milestone") }
```

Comment:
This is an indication of something being a bit premature - whilst this line is here we probably shouldn't be treating it as a releasable thing.